packages/lesswrong/components/editor/EditTitle.tsx

```diff
@@ -62,8 +70,8 @@ const EditTitle = ({document, value, path, placeholder, updateCurrentValues, cla
   }, [document, updatePost, lastSavedTitle, flash])
 
   return <Input
-    className={classNames(classes.root, {[classes.question]: question})}
-    placeholder={ question ? "Question Title" : placeholder }
+    className={classNames(classes.root, {[classes.question]: question && !isEAForum})}
+    placeholder={displayPlaceholder}
```

Comment:
The one thing I didn't do is put the link icon in the title placeholder. It takes a string input and I didn't think it was worth working around that